### PR TITLE
🚧Review and fix documentation 

### PR DESCRIPTION
**Pull request by kselena**
<h1 style="font-weight:bold" align="center">Preview Feature Pull Request</h1>
<h2 style="font-weight:bold" align="center">✅Code Review Checklist✅</h2>

- [x] The **_head(source)_** branch for this pull request is a **_preview feature_** branch, with the correct naming syntax, in the following example:
  - A **_head(source)_** branch is the branch being merged into a target branch.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/feature/<issue-num>-<description>
      Example: preview/feature/123-my-preview-feature
      ```
    </details>

- [x] The **_head(source)_** branch for this pull request is created from a **_preview release_** branch, with the correct naming syntax, in the following example:
  <details closed><summary>Example</summary>

    ``` xml
    Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
    Example: preview/v1.2.3-preview.4
    ```
  </details>

- [x] The **_base(target)_** branch for this pull request is a **_preview release_** branch, with the correct naming syntax, in the following example:
  - A **_base(target)_** branch is the branch that the **_head(source)_** branch is merging into.

    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
      Example: preview/v1.2.3-preview.4
      ```
    </details>

💡For more information on branching, refer to the project's [branching documentation](../../Documentation/Branching.md).

- [x] Associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [x] Pull request title matches the title of the associated issue.

- [x] A `preview` label has been added to the pull request.

- [x] The labels attached to this PR match the labels attached to the associated issue.

- [x] Project assigned matches the project of the associated issue.

- [x] Milestone assigned matches the milestone of the associated issue.

- [x] My code follows the coding style of this project.
  - The style is enforced by the **_.editorconfig_** files in the project and displayed as warnings.  If there is an edge case, reach out and let's discuss it.

- [x] I have manually tested my code changes to the best of my ability.

- [x] All tests passed locally.
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
  - Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
  - Status checks run for this pull request when merging into the **_preview/v\*.\*.\*-preview.\*_** branch.  These status checks run every time a change is pushed to the pull request.  These checks validate version syntax, tagging, builds, unit tests, and more.

### CODE REVIEW

File: CICD/ExtensionMethods.cs [hunk 1]
```diff
@@ -371,8 +371,8 @@ public static string GetVersion(this Project project)
 
         if (string.IsNullOrEmpty(version))
         {
-            // TODO: Create custom exception name MissingVersionException
-            // TODO: In the exception, explain how to set the version
+            // TODO: Create custom exception name MissingVersionException - **Do you still need this?**
```
Comment (KinsonDigitalAdmin) [inline CICD/ExtensionMethods.cs:374]: Yes, I still need these.  Please revert the changes that you made.

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.4.md [hunk 2]
```diff
@@ -13,15 +13,15 @@ As with all software, there is always a chance for issues and bugs, especially f
...
+   - `ExecutionContextService` - Determines whether or not the build is running locally or on the server which increases testability.
    - `GitHubClientService` - Creates a GitHubClient singleton with authorized requests.  This makes it much easier to be able to deal with the `GitHubClient` locally and on the server.
    - `JsonService` - Serializes and deserializes JSON data using abstraction to increase testability.
-   - `SecretService` - Loads secrets locally or on the server depending on where the build is running.
+   - `SecretService` - Loads secrets locally or on the server depending where the build is running.
 2. [#19](https://github.com/KinsonDigital/CICD/issues/19) - Created factories to help generate HTTP clients and tokens.
-   - `HttpClientFactory` - Abstracts away the process of creating HTTP clients.  Increases testability.
-   - `TokenFactory` - Abstracts the retrieval of a token depending on if the build is running locally or on the server.
+   - `HttpClientFactory` - Abstracts away the process of creating HTTP clients which ncreases testability.
```
Comment (KinsonDigitalAdmin) [inline Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.4.md:23]: The new change is spelled incorrectly.

```suggestion
   - `HttpClientFactory` - Abstracts away the process of creating HTTP clients which increases testability.
```
